### PR TITLE
Change in Wazuh documentation Ansible

### PR DESCRIPTION
**Pull request by lastranikos**
## Description

Update on the proposed installation flow with Ansible. The flow should guide the installation of Wazuh with Opendistro.

This PR originates from the following issue:
  - https://github.com/wazuh/wazuh-ansible/issues/638

## Checks
- [ ] It compiles without warnings.
- [ ] Spelling and grammar. 
- [ ] Used impersonal speech. 
- [ ] Used uppercase only on nouns. 
- [x] Updated the `redirect.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).

### CODE REVIEW

File: source/deploying-with-ansible/guide/install-opendistro-server.rst [hunk 1]
```diff
@@ -0,0 +1,554 @@
...
+Using **Opendistro** and **Kibana** roles we will install and configure the Opendistro Stack server components, there are several variables we can use to customize the installation or configuration. To consult the default configuration go to this :ref:`section <wazuh_ansible_reference>`.
+
+If we want to change the default configuration we can change the following files:
+- ``/etc/ansible/roles/wazuh-ansible/roles/opendistro/opendistro-elasticsearch/defaults/main.yml``
+- ``/etc/ansible/roles/wazuh-ansible/roles/opendistro/opendistro-kibana/defaults/main.yml``
+
+We also can create another YAML file only with the content we want to change the configuration for each role. We can find more information here:
+
+- :ref:`Opendistro <opendistro-elasticsearch>` role.
+- :ref:`Kibana <opendistro-kibana>` role.
```
[the remark this answers is not in the document]
Comment (lastranikos) [reply source/deploying-with-ansible/guide/install-opendistro-server.rst]: Done

File: source/deploying-with-ansible/guide/install-opendistro-server.rst [hunk 2]
```diff
@@ -0,0 +1,554 @@
...
+--------------------------
+
+We could configure these three files and execute them, but we are going to create a single file that executes the installation of the services in our Elastic Stack Server.
+
+.. code-block:: console
+
+	ansible@ansible:/etc/ansible/wazuh-ansible$ cat playbooks/wazuh-opendistro-and-kibana.yml
+
+.. code-block:: yaml
+	:class: output
```
[the remark this answers is not in the document]
Comment (lastranikos) [reply source/deploying-with-ansible/guide/install-opendistro-server.rst]: Done

File: source/deploying-with-ansible/guide/install-opendistro-server.rst [hunk 3]
```diff
@@ -0,0 +1,565 @@
...
+
+.. meta::
+  :description: Deploying the Open Distro for Elasticsearch server involves installing the Open Distro and Kibana services. Find out how to do it step by step in this section.
+
+.. _wazuh_ansible_elk_server:
+
+Install Open Distro for Elasticsearch Server
+============================================
+
+The deployment of the Open Distro for Elasticsearch server involves the installation of Opendistro and Kibana services. In the repository of Ansible that Wazuh has we can find the playbooks and roles necessary to carry out the installation. The Ansible server must have access to the Open Distro for Elasticsearch server.
```
Comment (Arcapas) [inline source/deploying-with-ansible/guide/install-opendistro-server.rst:11]:
```suggestion
The deployment of the Open Distro for the Elasticsearch server involves the installation of Opendistro and Kibana services. In the repository of Ansible that Wazuh has, we can find the playbooks and roles necessary to carry out the installation. The Ansible server must have access to the Open Distro for Elasticsearch server.
```

File: source/deploying-with-ansible/roles/wazuh-opendistro.rst [hunk 4]
```diff
@@ -1,24 +1,24 @@
-.. Copyright (C) 2021 Wazuh, Inc.
+.. Copyright (C) 2020 Wazuh, Inc.
 
-.. _ansible-wazuh-elasticsearch:
+.. _ansible-wazuh-opendistro:
 
-Elasticsearch
---------------
+Open Dstro
```
Comment (Arcapas) [inline source/deploying-with-ansible/roles/wazuh-opendistro.rst]:
```suggestion
Open Distro
```

File: source/deploying-with-ansible/roles/wazuh-opendistro.rst [hunk 5]
```diff
@@ -38,6 +38,6 @@ Next, run the ansible playbook: ::
 
   $ ansible-playbook wazuh-elastic.yml -e@vars-production.yml
 
-The example above will install Elasticsearch and set the listen address to: ``10.1.1.10`` using ``vars-production.yml``.
+The example above will install Opendistro and set the listen address to: ``10.1.1.10`` using ``vars-production.yml``.
```
Comment (Arcapas) [inline source/deploying-with-ansible/roles/wazuh-opendistro.rst]:
```suggestion
The example above will install Open Distro and set the listen address to: ``10.1.1.10`` using ``vars-production.yml``.
```

**Review by Arcapas — COMMENTED** [review]
I left some review comments that need to be accepted.

File: source/deploying-with-ansible/roles/wazuh-opendistro.rst (hunk 4 above)
Comment (lastranikos) [reply source/deploying-with-ansible/roles/wazuh-opendistro.rst]: This suggestion is out of date. The requested change is already present:
  - https://github.com/wazuh/wazuh-documentation/pull/4479/files

File: source/deploying-with-ansible/roles/wazuh-opendistro.rst [hunk 6]
```diff
@@ -1,24 +1,24 @@
...
-.. _ansible-wazuh-elasticsearch:
+.. _ansible-wazuh-opendistro:
 
-Elasticsearch
---------------
+Open Dstro
+----------
 
-This role is intended to deploy Elasticsearch node, you have some variables that can be used to customize the installation:
+This role is intended to deploy Opendistro node, you have some variables that can be used to customize the installation:
```
[the remark this answers is not in the document]
Comment (lastranikos) [reply source/deploying-with-ansible/roles/wazuh-opendistro.rst]: This suggestion is out of date. The requested change is already present:
  - https://github.com/wazuh/wazuh-documentation/pull/4479/files

File: source/deploying-with-ansible/roles/wazuh-opendistro.rst (hunk 5 above)
Comment (lastranikos) [reply source/deploying-with-ansible/roles/wazuh-opendistro.rst]: This suggestion is out of date. The requested change is already present:
  - https://github.com/wazuh/wazuh-documentation/pull/4479/files

**Review by Arcapas — APPROVED** [review]
LGTM

**Review by rauldpm — APPROVED** [review]
LGTM

File: source/deploying-with-ansible/guide/install-opendistro-server.rst [hunk 7]
```diff
@@ -0,0 +1,565 @@
...
+	├── wazuh-odfe-single.yml
+	├── wazuh-opendistro-kibana.yml
+	└── wazuh-opendistro.yml
+
+
+Using **Opendistro** and **Kibana** roles, we will install and configure the Open Distro for Elasticsearch server components. There are several variables we can use to customize the installation or configuration. To consult the default configuration go to this :ref:`section <wazuh_ansible_reference>`.
+
+If we want to change the default configuration, we can change the following files:
+- ``/etc/ansible/roles/wazuh-ansible/roles/opendistro/opendistro-elasticsearch/defaults/main.yml``
+- ``/etc/ansible/roles/wazuh-ansible/roles/opendistro/opendistro-kibana/defaults/main.yml``
```
Comment (rauldpm) [inline source/deploying-with-ansible/guide/install-opendistro-server.rst:166]: Fix this, this must be show in different lines

image

**Review by rauldpm — CHANGES_REQUESTED** [review]
Build the documentation locally and make sure that all changes made to all documentation modified/created look as they should

File: source/deploying-with-ansible/guide/install-opendistro-server.rst (hunk 7 above)
Comment (lastranikos) [reply source/deploying-with-ansible/guide/install-opendistro-server.rst]: Done
image

**Review by rauldpm — APPROVED** [review]

File: source/deploying-with-ansible/roles/wazuh-opendistro.rst [hunk 8]
```diff
@@ -1,24 +1,24 @@
...
-.. _ansible-wazuh-elasticsearch:
+.. _ansible-wazuh-opendistro:
 
-Elasticsearch
---------------
+Open Distro
+-----------
 
-This role is intended to deploy Elasticsearch node, you have some variables that can be used to customize the installation:
+This role is intended to deploy Open Distro node, you have some variables that can be used to customize the installation:
```
Comment (s-ocando) [inline source/deploying-with-ansible/roles/wazuh-opendistro.rst:8]:
```suggestion
This role is intended to deploy the Open Distro node. The following variables can be used to customize the installation:
```

File: source/deploying-with-ansible/guide/install-opendistro-server.rst [hunk 9]
```diff
@@ -0,0 +1,566 @@
...
+
+.. meta::
+  :description: Deploying the Open Distro for Elasticsearch server involves installing the Open Distro and Kibana services. Find out how to do it step by step in this section.
+
+.. _wazuh_ansible_elk_server:
+
+Install Open Distro for Elasticsearch Server
+============================================
+
+The deployment of the Open Distro for the Elasticsearch server involves the installation of Opendistro and Kibana services. In the repository of Ansible that Wazuh has, we can find the playbooks and roles necessary to carry out the installation. The Ansible server must have access to the Open Distro for Elasticsearch server.
```
Comment (s-ocando) [inline source/deploying-with-ansible/guide/install-opendistro-server.rst:11]:
```suggestion
The deployment of the Open Distro for the Elasticsearch server involves the installation of Open Distro for Elasticsearch and Kibana services. In the Wazuh Ansible repository, we can find the playbooks and roles necessary to carry out the installation. The Ansible server must have access to the Open Distro for Elasticsearch server.
```

File: source/deploying-with-ansible/guide/install-opendistro-server.rst [hunk 10]
```diff
@@ -0,0 +1,566 @@
...
+
+.. warning::
+
+	In previous versions of this guide, playbooks were used pointing to roles to install Elasticsearch.
+	Currently, the roles for the installation following guides from previous versions are preserved.
+	Since version 4.2.4 the installation steps of this guide point to the roles that install Open Distro for Elasticsearch.
+
+.. warning::
+
+    About the nomenclatures:
```
Comment (s-ocando) [inline source/deploying-with-ansible/guide/install-opendistro-server.rst:26]:
```suggestion
    About the nomenclature:
```

File: source/deploying-with-ansible/guide/install-opendistro-server.rst [hunk 11]
```diff
@@ -0,0 +1,566 @@
...
+- `1 - Access to wazuh-ansible`_
+- `2 - Preparing the playbook`_
+- `3 - Running the playbook`_
+
+
+.. warning::
+
+	In previous versions of this guide, playbooks were used pointing to roles to install Elasticsearch.
+	Currently, the roles for the installation following guides from previous versions are preserved.
+	Since version 4.2.4 the installation steps of this guide point to the roles that install Open Distro for Elasticsearch.
```
Comment (s-ocando) [inline source/deploying-with-ansible/guide/install-opendistro-server.rst:22]:
```suggestion
	Starting Wazuh v4.2.5, the installation steps of this guide point to the roles that install Open Distro for Elasticsearch.
```

File: source/deploying-with-ansible/guide/install-opendistro-server.rst [hunk 12]
```diff
@@ -0,0 +1,566 @@
...
+
+- `1 - Access to wazuh-ansible`_
+- `2 - Preparing the playbook`_
+- `3 - Running the playbook`_
+
+
+.. warning::
+
+	In previous versions of this guide, playbooks were used pointing to roles to install Elasticsearch.
+	Currently, the roles for the installation following guides from previous versions are preserved.
```
Comment (s-ocando) [inline source/deploying-with-ansible/guide/install-opendistro-server.rst:21]:
```suggestion
	Currently, the roles for the installation following previous guides are preserved.
```

**Pull request merged**